modules/planning/navi/decider/navi_path_decider.cc

```diff
@@ -240,7 +240,8 @@ bool NaviPathDecider::GetBasicPathData(
   double min_path_len = config_.min_path_length();
   // get min path plan length s = v0 * t + 1 / 2.0 * a * t^2
   double path_len = start_plan_v_ * config_.min_look_forward_time() +
-                    start_plan_a_ * pow(config_.min_look_forward_time(), 2) / 2.0;
+                    start_plan_a_ *
+                    pow(config_.min_look_forward_time(), 2) / 2.0;
```

Comment:
@Capri2014 Lint issue has been fixed here.

Reply:
@Capri2014 The Build Failed issue below is caused by `apollo.sh test`, not lint.
[5921.5](https://travis-ci.com/ApolloAuto/apollo/jobs/201384753#L1790-L1791) "JOB=citest_extended""JOB=citest_extended" | failed